slides.md

```diff
@@ -501,6 +501,76 @@ export default async function customer(fastify) {
 
 ---
 
+# A04 Exercise: a bot prevention mechanism
+
+<div class="dense">
+
+- The `/buy-product` endpoint is not designed to prevent a bot attack to buy products.
+- It means that an attack can be generated to buy too many products in a short period of time
```

Comment:
I assume this scenario is meant to be about scalpers buying before other people can buy?

If so it might be worth explaining in the slides why that's an issue and what's the business problem (ie. someone can buy a lot of stock quickly and leave legitimate customers without any)

Reply:
Hi Liana, thanks for your review!

Maybe using "bot attack" terminology wasn't really the best idea.

> I assume this scenario is meant to be about scalpers buying before other people can buy?

The idea is to reproduce exactly this scenario. I will improve the text to explain the business problem better.